apollo-router/src/axum_http_server_factory.rs

```diff
@@ -130,22 +130,6 @@ where
         );
     }
 
-    endpoints.insert(
```

Comment:
Suggestion: maybe we don't remove this but replace the response with a 404 instructing the user to change their health endpoint?

Reply:
That's a really good idea.

Reply:
@BrynCooke How do you think I should do this?  The configuration is no longer there but we used to let users configure it manually and we changed to to `/health` in a recent PR.  They should start receiving configuration warnings if they've manually overrode it, so should I just do it for the endpoint that **used** to be our default (`/.well-known/apollo/server-health`)?  (Only on the main port?)

Reply:
Nevermind, figured it out in f8c74c9e707219efee78082feb06c61b